openedx/features/survey_report/context_processors.py

```diff
@@ -0,0 +1,34 @@
+"""
+This is the survey report contex_processor modules
+
+This is meant to determine the visibility of the survey report banner
+across all admin pages in case a survey report has not been generated
+
+"""
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta  # for months test
+from .models import SurveyReport
+from django.urls import reverse
+from django.conf import settings
+
+
+def admin_extra_context(request):
+    """
+    This function sends extra context to every admin site
+
+    The current treshhold to show the banner is one month but this can be redefined in the future
+
+    """
+    months = settings.SURVEY_REPORT_CHECK_THRESHOLD
```

Comment:
I'd like to request the ability to turn this banner off entirely. Some organizations/operators may not feel comfortable sharing this data back and there doesn't currently seem to be any ability to disable it entirely. Please correctly me if I'm wrong.

From what I gather we can set the `SURVEY_REPORT_ENDPOINT` to a bogus endpoint but the banner will still appear. We could set the `SURVEY_REPORT_CHECK_THRESHOLD` to a very large number, but the banner will still appear until one report is sent. There seems to be no way to disable this entirely

Reply:
Hello @georgebabey, you are right, completely disabling is not present at the moment. It was in the first draft and with the many reviews it was sidetracked. Still it is a completely reasonable request. We'll include that in master and add it to the backport.